### PR TITLE
Use raw ores for iron and gold lumps in Mineclone

### PR DESCRIPTION
**Pull request by programmerjake**
Fixes: #341

### CODE REVIEW

File: technic/mod.conf [hunk 1]
```diff
@@ -1,3 +1,3 @@
 name = technic
 depends = pipeworks, technic_worldgen, basic_materials
-optional_depends = mcl_core, mcl_sounds, default, bucket, mesecons, mesecons_mvps, digilines, digiline_remote, unified_inventory, dye, craftguide, i3, mtt, vizlib, moreores, mcl_buckets, mcl_explosions, mcl_craftguide, zzzz_exchangeclone_init
+optional_depends = mcl_core, mcl_sounds, default, bucket, mesecons, mesecons_mvps, digilines, digiline_remote, unified_inventory, dye, craftguide, i3, mtt, vizlib, moreores, mcl_raw_ores, mcl_buckets, mcl_explosions, mcl_craftguide, zzzz_exchangeclone_init
```
Comment (OgelGames) [inline technic/mod.conf:3]: This is not needed.
```suggestion
optional_depends = mcl_core, mcl_sounds, default, bucket, mesecons, mesecons_mvps, digilines, digiline_remote, unified_inventory, dye, craftguide, i3, mtt, vizlib, moreores, mcl_buckets, mcl_explosions, mcl_craftguide, zzzz_exchangeclone_init
```

**Review by OgelGames — CHANGES_REQUESTED** [review]

File: technic/mod.conf (hunk 1 above)
Comment (programmerjake) [reply technic/mod.conf:3]: why not? don't we want `mcl_raw_ores` to be loaded before `technic` so it doesn't try to use a bunch of items that don't yet exist?

Comment (OgelGames) [reply technic/mod.conf:3]: Because they are only used in recipes, which don't require the items to exist.

Comment (programmerjake) [reply technic/mod.conf]: ok, i removed that commit

**Review by OgelGames — APPROVED** [review]

**Pull request merged**